### DISCUSSION TITLE
PVE 的虚拟机运行的，不能获取 IPV6 地址，所以不能使用 IPV6 的节点，能不能支持IPV6 ？

### DISCUSSION THREAD

**Discussion by paranoiagu** [kkkgo/PaoPaoGateWay#29] [想法]
现在有些节点只支持 ipv6

**Comment by kkkgo** [comment]
https://github.com/kkkgo/PaoPaoGateWay/discussions/10

**Reply by kkkgo** [reply]
相关功能已更新，查看最新release：https://github.com/kkkgo/PaoPaoGateWay/releases/
